### DISCUSSION TITLE
Wallet connect inside an iframe

### DISCUSSION THREAD

**Discussion by manish252488** [WalletConnect/walletconnect-monorepo#3793] [Developer Support]
I have integrated a wallet connected inside an iframe. when the iframe is launched in ios, Wallet Connect is not able to detect any wallet that's installed in the mobile. it shows the cross origin error 
Blocked a frame with origin "https://my-iframe.domain.net" from accessing a frame with origin "https://verify.walletconnect.com" Protocols, domains, and ports must match.

**Comment by guotie** [comment]
same problem, any solutions?

**Reply by manish252488** [reply]
No solution !! we had to remove  the iframe.
